lib/app.js

```diff
@@ -212,11 +219,11 @@ function extractFields (parsedData) {
 //   };
 
 // }
-// function descriptionText (description) {
-//   // preserve the line breaks when converting to text
-//   const html = '<div>' + description.html().replace(/<\/p>/g, '\n</p>') + '</div>';
-//   return cheerio.load(html)('div').text();
-// }
+function descriptionText (description) {
+  // preserve the line breaks when converting to text
+  const html = cheerio.load('<div>' + description.replace(/\<br\>/g, '\r\n') + '</div>');
```

Comment:
any particular reason to replace `br` instead of `p`? I honestly don't remember why it was p. I guess the important thing is that you make sure the output text is what we expect in terms of line breaks. (also I think sometimes the breaks are plain `<br>`. I'd say just pick a couple of example apps and see what's in their description.

Reply:
I also didn't understand why was `p` there... I tested with some apps, and got only `<br>`. I'll test with a detailed list to see if it varies.

Reply:
Did a quick test with 10 apps, all seems to be good. 

You can check: 

```
play.list({
  category: gplay.category.GAME_ACTION,
  collection: gplay.collection.TOP_FREE,
  num: 10,
  fullDetail: true,
})
  .then(data => data.forEach(x => console.log(x.description)))
```